src/graph/planner/ngql/PathPlanner.cpp

```diff
@@ -183,9 +183,11 @@ StatusOr<SubPlan> PathPlanner::allPathPlan() {
   auto withProp = pathCtx_->withProp;
   auto* path = AllPaths::make(
       qctx, leftPlan.root, rightPlan.root, pathCtx_->space.id, steps, pathCtx_->noLoop, withProp);
-  auto vertexProp = SchemaUtil::getAllVertexProp(qctx, pathCtx_->space.id, withProp);
```

Comment:
Any tests?

Reply:
already exist in allpath.feature

Reply:
But it doesn't detect these different of results.

Reply:
the tester has its own comparison function, which is a per-character comparison